### PR TITLE
Added 'Database' feature to 'BACKEND'

### LINKED ISSUE #597: [Feature] "Database" subcategory for "Backend" category
### Description

LinksHub is currently ***missing*** an important category dedicated to various ***database solutions*** such as MongoDB, PostgreSQL, Redis, and more. 

***Databases*** play a crucial role in powering the infrastructure of most websites. They store and manage vast amounts of data, ensuring efficient retrieval and manipulation. As such, it is essential to have a ***dedicated subcategory*** that focuses on the resources available for the various diverse database solutions.

### Screenshots
A mockup of what I want to achieve
Screenshot 2023-05-21 at 12 21 47

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Rutujeet**
Closes #597 

Added 'Database' feature to 'BACKEND' as requested

Screenshot 2023-05-22 at 12 50 49 PM

### CODE REVIEW

**Review by CBID2 — DISMISSED** [review]
Looks great @Rutujeet! :) LGTM! :)

**Comment by rupali-codes** [comment]
@Rutujeet Please resolve the conflicts carefully and don't **Close** the PR. You can learn how to resolve conflicts in [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

**Comment by CBID2** [comment]
@Rutujeet, please resolve the branch conflicts

**Comment by Rutujeet** [comment]
> hey, could you please resolve merge conflicts?

I'll resolve it asap, I'm away

File: database/backend/database.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "MongoDB",
+    "description": "MongoDB is a source-available cross-platform document-oriented NoSQL database program",
+    "url": "https://www.mongodb.com/",
+    "category": "backend",
+    "subcategory": "database",
+    "language": "english"
```
Comment (rupali-codes) [inline database/backend/database.json:8]:
```suggestion
```

File: database/backend/database.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "name": "MongoDB",
+    "description": "MongoDB is a source-available cross-platform document-oriented NoSQL database program",
```
Comment (CBID2) [inline database/backend/database.json:4]: [Proofreading]
```suggestion
    "description": "MongoDB is a source cross-platform document-oriented NoSQL database program",
```

**Comment by rupali-codes** [comment]
@CBID2 what did you do 😂

File: database/backend/database.json [hunk 3]
```diff
@@ -5,6 +5,5 @@
     "url": "https://www.mongodb.com/",
     "category": "backend",
     "subcategory": "database",
```
Comment (rupali-codes) [inline database/backend/database.json:7]:
```suggestion
    "subcategory": "database"
```

**Comment by CBID2** [comment]
> @CBID2 what did you do 😂

I made a commit @rupali-codes. Proofreading issue.

**Comment by rupali-codes** [comment]
> > @CBID2 what did you do 😂
> 
> I made a commit @rupali-codes. Proofreading issue.

Oh okay

**Review by CBID2 — CHANGES_REQUESTED** [review]
So close. Still needs more improvement.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
lgtm too

**Comment by CBID2** [comment]
> > > @CBID2 what did you do 😂
> 
> > 
> 
> > I made a commit @rupali-codes. Proofreading issue.
> 
> 
> 
> Oh okay

- I think we need another review from you @rupali-codes.

**Pull request merged**